app/models/profile.rb

```diff
@@ -34,7 +21,23 @@ def activate
     end
   end
 
-  def verified?
-    verified_at.present?
+  def plain_pii
+    @_plain_pii ||= Pii::Attributes.new
```

Comment:
that's a good point. I am not happy with how the PII encryption is neither automatic in a hook, nor clearly a manual process.

We could get rid of `plain_pii` altogether to remove confusion. E.g.

```ruby
profile = Profile.new( user: user )
pii_attrs = Pii::Attributes.new_from_hash( ssn: '1234' )
profile.encrypt_pii(passphrase, pii_attrs)
profile.save!

pii_decrypted = profile.decrypt_pii(passphrase)
puts pii_decrypted == pii_attrs  # -> true
```

You would just never handle un-encrypted PII aside from an explicit `Pii::Attributes` instance. The `Profile` only ever holds it encrypted.

That would make the tests a little more verbose but at least there would not be unexpected behaviour.

Thoughts?